src/Umbraco.Web.UI.Client/src/views/packages/views/installed.html

```diff
@@ -17,6 +17,7 @@
                             </div>
                             <div class="umb-package-list__item-content">
                                 <div class="umb-package-list__item-name">{{ installedPackage.name }}</div>
+                                <div class="umb-package-list__item-id" ng-if="installedPackage.id">{{ installedPackage.id }}</div>
```

Comment:
I think just showing the ID in the lighter grey will be clear enough, so we don't have to prefix it with `ID:`, but it might look a little redundant if the name and ID are the same...